managed-databases/postgresql-and-mysql/how-to/manage-manual-backups.mdx

```diff
@@ -17,40 +17,52 @@ categories:
 Manual backups are useful in cases when you need to have a backup of a specific state of the Database Instance. They can be used for Database Instances with local storage.
 
 <Macro id="iam-requirements" />
- 
+
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have a [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart)
 </Message>
 
 ## How to create manual backups
 
-1. Click the **Backups** tab on your selected database's information page.
+1. Click the **Backups** tab on your selected Database Instance's information page.
 2. Click **Create backup**.
 3. Enter a **name** for the backup, choose the database and set an expiration date for the backup to be removed automatically.
     <Message type="note">
       You can click <Icon name="more" /> > **Edit** to change the name and expiration anytime.
     </Message>
 4. Click **Create backup** to launch the backup process for the database.
 
-## How to restore manual backups
+## How to restore backups
 
-1. Click the **Backups** tab on your selected database's information page.
-2. Click <Icon name="more" /> next to the name of the backup you want to restore.
-3. Click **Restore backup**.
+1. Click the **Backups** tab on your selected Database Instance's information page.
+2. Click <Icon name="more" /> > **Restore** next to the name of the backup you want to restore. A pop-up displays.
+4. Select the Database Instance and the database to which you wish to restore the backup.
+5. Click **Restore backup**.
     <Message type="note">
       Large database backups can take up to several hours to restore.
     </Message>
 
-## How to delete manual backups
+## How to export backups
+
+1. Click the **Backups** tab on your selected Database Instance's information page.
+2. Click <Icon name="more" /> next to the name of the backup you want to export.
+3. Click **Prepare export**. Once the export is prepared, the **Download** button displays.
+    <Lightbox src="scaleway-download-export-button.webp" alt="" />
+4. Click **Download**.
+    <Message type="note">
+      Backups are exported in `.gz` format.
```

Comment:
This is not true. 

PostgreSQL backups are exported in PostgreSQL compress format. 
MySQL backups are compressed with gzip